app/data/schedules.js

```diff
@@ -1,4 +1,6 @@
-// module.exports = [];
+module.exports = {
+}
```

Comment:
As it stands, this line is just replicating the default value of `module.exports`.